src/lib/Menu/Menu.svelte

```
@@ -43,12 +44,20 @@
   }
 
-  function openMenu(startIndex: number = 0) {
+  function openMenu(startIndex: number | null = null) {
     open = true;
-    focusedIndex = startIndex;
+    // With a known selection, opening focuses the selected option (listbox
+    // convention) instead of always parking the focus highlight on item 0.
+    const selectedItem =
+      selectedValue != null
+        ? items.find((item) => item.value === selectedValue && item.disabled !== true)
+        : undefined;
```

ESLint

[error] undefined is not allowed. Use null or proper type checking instead.

(no-restricted-syntax)